test/controllers/InstalmentsControllerSpec.scala

```diff
@@ -130,4 +133,19 @@ class InstalmentsControllerSpec extends ItSpec {
     }
   }
 
+  "InstalmentsController.retrieveInstalmentOptions" - {
+    "should filter out any instalments that are less than £1" in {
+      val plansIncludingOneLessThanAPound: List[PaymentPlan] = List(
+        TdAll.paymentPlan(1, amountDue = AmountDue(AmountInPence(1))),
+        TdAll.paymentPlan(2, amountDue = AmountDue(AmountInPence(99))),
+        TdAll.paymentPlan(3),
+        TdAll.paymentPlan(4),
+        TdAll.paymentPlan(5),
```

Comment:
maybe clearer to make the `amountDue` values explicit here

Reply:
Sure